jq-defect-2020-1/mayhem/jq_parse_fuzzer/Mayhemfile

```diff
@@ -0,0 +1,10 @@
+version: '1.2'
+project: jq-defect-2020-1
+target: fuzzer_jq_pair 
+baseimage: forallsecure/jq-defect-2020-1
+duration: 600
```

Comment:
I would double the time it took to find on staging etc. to make sure it is found in one run